Debian 13 stable - missing gtk4-session-lock.h stopping wayfire compile

## Wayfire version
https://github.com/WayfireWM/wayfire git clone of code

## Describe the bug
No way to install gtk4-session-lock on Debian 13 stable

## To Reproduce
Steps to reproduce the behavior:

1. Compile from git clone of source with wf-install script
2. Compile fails with ../src/locker/locker.cpp:13:10: fatal error: gtk4-session-lock.h: No such file or directory
3. I can't find a way to install it.

## Expected behavior
Wayfire to compile on Debian 14

## Additional context
[68/95] Compiling C++ object src/locker/wf-locker.p/locker.cpp.o
FAILED: src/locker/wf-locker.p/locker.cpp.o 
ccache c++ -Isrc/locker/wf-locker.p -Isrc/locker -I../src/locker -Isrc/util -I../src/util -Isubprojects/wf-json -I../subprojects/wf-json -Isubprojects/gvc -I../subprojects/gvc -I/usr/include/gtk4-layer-shell -I/usr/include/gtk-4.0 -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/sysprof-6 -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/x86_64-linux-gnu -I/usr/include/webp -I/usr/include/graphene-1.0 -I/usr/lib/x86_64-linux-gnu/graphene-1.0/include -I/usr/include/gtkmm-4.0 -I/usr/lib/x86_64-linux-gnu/gtkmm-4.0/include -I/usr/include/giomm-2.68 -I/usr/lib/x86_64-linux-gnu/giomm-2.68/include -I/usr/include/glibmm-2.68 -I/usr/lib/x86_64-linux-gnu/glibmm-2.68/include -I/usr/include/sigc++-3.0 -I/usr/lib/x86_64-linux-gnu/sigc++-3.0/include -I/usr/include/gio-unix-2.0 -I/usr/include/cairomm-1.16 -I/usr/lib/x86_64-linux-gnu/cairomm-1.16/include -I/usr/include/pangomm-2.48 -I/usr/lib/x86_64-linux-gnu/pangomm-2.48/include -I/usr/include/gtk-4.0/unix-print -I/usr/share/wayfire/include -I/usr/include/libevdev-1.0/ -I/usr/include/libxml2 -fdiagnostics-color=always -D_GLIBCXX_ASSERTIONS=1 -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c++17 -O2 -g -DHAVE_PULSE=1 -Wno-pedantic -Wno-unused-parameter -Wno-parentheses '-DICONDIR="/usr/share/wayfire/share/wayfire/icons"' '-DRESOURCEDIR="/usr/share/wayfire/share/wayfire"' '-DMETADATA_DIR="/usr/share/wayfire/share/wayfire/metadata/wf-shell"' '-DSYSCONF_DIR="/usr/share/wayfire/etc"' -pthread -D_REENTRANT -mfpmath=sse -msse -msse2 -mfpmath=sse -msse -msse2 -MD -MQ src/locker/wf-locker.p/locker.cpp.o -MF src/locker/wf-locker.p/locker.cpp.o.d -o src/locker/wf-locker.p/locker.cpp.o -c ../src/locker/locker.cpp
../src/locker/locker.cpp:13:10: fatal error: gtk4-session-lock.h: No such file or directory
   13 | #include <gtk4-session-lock.h>
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[69/95] Compiling C++ object src/locker/wf-locker.p/lockscreen.cpp.o
FAILED: src/locker/wf-locker.p/lockscreen.cpp.o 
ccache c++ -Isrc/locker/wf-locker.p -Isrc/locker -I../src/locker -Isrc/util -I../src/util -Isubprojects/wf-json -I../subprojects/wf-json -Isubprojects/gvc -I../subprojects/gvc -I/usr/include/gtk4-layer-shell -I/usr/include/gtk-4.0 -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/sysprof-6 -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/x86_64-linux-gnu -I/usr/include/webp -I/usr/include/graphene-1.0 -I/usr/lib/x86_64-linux-gnu/graphene-1.0/include -I/usr/include/gtkmm-4.0 -I/usr/lib/x86_64-linux-gnu/gtkmm-4.0/include -I/usr/include/giomm-2.68 -I/usr/lib/x86_64-linux-gnu/giomm-2.68/include -I/usr/include/glibmm-2.68 -I/usr/lib/x86_64-linux-gnu/glibmm-2.68/include -I/usr/include/sigc++-3.0 -I/usr/lib/x86_64-linux-gnu/sigc++-3.0/include -I/usr/include/gio-unix-2.0 -I/usr/include/cairomm-1.16 -I/usr/lib/x86_64-linux-gnu/cairomm-1.16/include -I/usr/include/pangomm-2.48 -I/usr/lib/x86_64-linux-gnu/pangomm-2.48/include -I/usr/include/gtk-4.0/unix-print -I/usr/share/wayfire/include -I/usr/include/libevdev-1.0/ -I/usr/include/libxml2 -fdiagnostics-color=always -D_GLIBCXX_ASSERTIONS=1 -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c++17 -O2 -g -DHAVE_PULSE=1 -Wno-pedantic -Wno-unused-parameter -Wno-parentheses '-DICONDIR="/usr/share/wayfire/share/wayfire/icons"' '-DRESOURCEDIR="/usr/share/wayfire/share/wayfire"' '-DMETADATA_DIR="/usr/share/wayfire/share/wayfire/metadata/wf-shell"' '-DSYSCONF_DIR="/usr/share/wayfire/etc"' -pthread -D_REENTRANT -mfpmath=sse -msse -msse2 -mfpmath=sse -msse -msse2 -MD -MQ src/locker/wf-locker.p/lockscreen.cpp.o -MF src/locker/wf-locker.p/lockscreen.cpp.o.d -o src/locker/wf-locker.p/lockscreen.cpp.o -c ../src/locker/lockscreen.cpp
In file included from ../src/locker/lockscreen.cpp:2:
../src/locker/locker.hpp:5:10: fatal error: gtk4-session-lock.h: No such file or directory
    5 | #include <gtk4-session-lock.h>
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[77/95] Compiling C++ object src/locker/wf-locker.p/timedrevealer.cpp.o
ninja: build stopped: subcommand failed.

--- a/meson.build
+++ b/meson.build
@@ -20,7 +20,10 @@ wayland_protos = dependency('wayland-protocols')
 gtkmm = dependency('gtkmm-4.0', version: '>=4.12')
 wfconfig = dependency('wf-config', version: '>=0.7.0') #TODO fallback submodule
 epoxy = dependency('epoxy')
-gtklayershell = dependency('gtk4-layer-shell-0', fallback: ['gtk4-layer-shell'])
+gtklayershell = dependency('gtk4-layer-shell-0', version: '>=1.3.0', required: false)
+if not gtklayershell.found()
+    gtklayershell = dependency('gtk4-layer-shell', fallback: ['gtk4-layer-shell'])
+endif
 libpulse = dependency('libpulse', required: get_option('volume-widget'))
 libgvc = subproject('gvc', default_options: ['static=true'], required: get_option('volume-widget'))
 pipewire = dependency('libpipewire-0.3', required: get_option('wp-mixer-widget'))

